spec/mongo/bulk_write_spec.rb

```diff
@@ -55,6 +55,19 @@
         end
       end
 
+      context 'when providing no requests' do
+
+        let(:requests) do
+          []
+        end
+
+        it 'raises an exception' do
+          expect {
+            bulk_write.execute
+          }.to raise_error(ArgumentError)
```

Comment:
I would add the expected exception message as the second argument, to ensure that the expected exception is produced (ArgumentError is a common exception and can be triggered elsewhere).

Reply:
done